pkg/apis/messaging/v1alpha1/sequence_validation.go

```diff
@@ -41,7 +39,7 @@ func (ps *SequenceSpec) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
-	if equality.Semantic.DeepEqual(ps.ChannelTemplate, eventingduck.ChannelTemplateSpec{}) {
+	if ps.ChannelTemplate == nil {
```

Comment:
What happens if channelTemplate is set but empty?

Reply:
you mean something like:
```yaml
spec:
  channelTemplate: {}
```
It will fail because we validate en the next lines wether it has APIVersion and Kind.

Reply:
Ahh, it was below the fold, so I didn't notice it.